skimage/color/colorlabel.py

```diff
@@ -113,7 +113,7 @@ def label2rgb(
         `bg_color` is `None`, and `kind` is `overlay`,
         background is not painted by any colors.
     bg_color : str or array, optional
-        Background color. Must be a name in `color_dict` or RGB float values
+        Background color. Must be a name in 'color_dict' or RGB float values
```

Comment:
I think this should be 

```suggestion
        Background color. Must be a name in :obj:`~.color_dict` or RGB float values
```
Rather than mute this error, we should make sure that `color_dict` appears in our HTML documentation. It's a bit awkward to tell users go lookup what you can pass to this parameter in x, when x isn't anywhere to be found...

Reply:
Your proposed fix unfortunately does not fix the issue. I have spent a good amount of time trying different options, but at the end, none of them worked. There are two cases like that (where we are trying to refer to a Python dict in the documentation) - this one and https://github.com/scikit-image/scikit-image/blob/8bebbbcda51a3574840c0d0cab2aa82535329bd7/skimage/io/manage_plugins.py#L176 .  I think we have two options here - either changing dicts into (data?)classes or simplifying the docs (as currently done). I would be happy to leave this to another PR.

Reply:
Okay, then my suggestion would be to turn this into
```
``skimage.color.color_dict``
```
for now. At least that tells users where to find it.